pkg/controller/usersignupcleanup/usersignup_cleanup_controller.go

```diff
@@ -79,7 +79,7 @@ func (r *Reconciler) Reconcile(request reconcile.Request) (reconcile.Result, err
 
 	// TODO remove this after migration complete
 	// Migrate the Approved property
-	if instance.Spec.Approved && !states.Approved(instance) {
+	/*if instance.Spec.Approved && !states.Approved(instance) {
```

Comment:
We can delete this block now instead of putting it in a comment, right?

Reply:
Done